docs/proxy.md

```
@@ -0,0 +1,383 @@
+### 2. Reaching the shop from the Windows browser
+
+Windows does not use WSL's resolver or trust store, so a browser on Windows needs
+two things the CLI cannot do from inside the subsystem (it prints them via
+`WSLWindowsAccessGuidance`):
+
+- **Trust the CA.** In WSL, copy the CA out to Windows:
+  ```bash
+  cp ~/.local/share/mkcert/rootCA.pem /mnt/c/Users/Public/shopware-cli-rootCA.pem
+  ```
+  Then, in a Windows terminal opened **as Administrator**, import it:
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)